### ISSUE TITLE
Picture and button press

### ISSUE THREAD

**Issue by DawnDuskStar** [ElijahWilde/buttons580#1]
I like the idea. But here are some suggestions. Once the picture was loaded, there was no way to center or increase the image. Also, I would like to see if there is a way to increase the button size, so it is easier to go and press. I did not see too much of a change with the press of a button, so I would not think it would be a bad idea to add sound for a pressure of a button to let the teacher know that there was button pressed. I still think it is a good start.

**Comment by sdimos** [comment]
I agree with DawnDuskStar's comments. I think adjusting the button size, being able to adjust the orientation of the image, and increasing the visual effect of when a button is pressed would be good additions to make this a super useful application. Additionally, my computer didn't read aloud any of the text in the buttons - which I think would be a very useful feature!

**Comment by obglass** [comment]
I agree with the above comments and I think this project is a great idea! One small thing i noticed as well is that when you make one button and adjust the text weight and height and go to make another one, the weight and height bars are at the same place as the previous one you made, but that doesn't reflect the size that it is on the button unless you move it - if that makes sense?

**Comment by DanielDanZhu** [comment]
Love the idea behind this project. One thing I'd say is that I would like a bit more a guide on how exactly to implement your button in my own web pages.
